rocketpy/simulation/flight.py

```diff
@@ -1911,7 +1911,7 @@ def u_dot_parachute(self, t, u, post_processing=False):
         rho = self.env.density.get_value_opt(u[2])
         to = 1.2
         ma = ka * rho * (4 / 3) * np.pi * R**3
-        mp = self.rocket.mass
+        mp = self.rocket.dry_mass
```

Comment:
I understand that a code might be slightly out of scope of a hotfix, but I advise improving the variable naming here for better clarity.

Reply:
I agree. This method is hard to read. 

Can we do that in a separate PR please? The Flight class is really old...